### PR TITLE
refactor(tile): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

changes to font size based on Figma.

### CODE REVIEW

File: src/components/calcite-tile/calcite-tile.scss [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+    text-color-3
+    inline-block
+    duration-150 
+    ease-in-out
+    select-none;
 }
 :host(:not([embed])) {
+  @apply p-3;
   box-shadow: 0 0 0 1px var(--calcite-ui-border-2);
   max-width: 300px;
```
Comment (jrowlingson) [inline src/components/calcite-tile/calcite-tile.scss:13]: Did we settle on `20rem` elsewhere? Could we do that here?

File: src/components/calcite-tile/calcite-tile.scss [hunk 2]
```diff
@@ -24,41 +23,43 @@ $spacing: $baseline/2;
   padding: unset;
```
Comment (jrowlingson) [inline src/components/calcite-tile/calcite-tile.scss:23]: p-0

File: src/components/calcite-tile/calcite-tile.scss [hunk 3]
```diff
@@ -24,41 +23,43 @@ $spacing: $baseline/2;
...
-  color: var(--calcite-ui-text-2);
-  transition: $transition;
-  font-weight: 500;
-  pointer-events: none;
+  @apply text-1-wrap
+  text-color-2
+  duration-150
+  ease-in-out
+  font-medium
+  pointer-events-none;
```
Comment (jrowlingson) [inline src/components/calcite-tile/calcite-tile.scss:37]: Indent

File: src/components/calcite-tile/calcite-tile.scss [hunk 4]
```diff
@@ -24,41 +23,43 @@ $spacing: $baseline/2;
...
 }
 .large-visual {
-  justify-items: center;
-  min-height: 200px;
+  @apply items-center;
+  min-height: theme("spacing.48");
```
Comment (jrowlingson) [inline src/components/calcite-tile/calcite-tile.scss:41]: :heart:

**Review by jrowlingson — APPROVED** [review]

**Comment by julio8a** [comment]
@bstifle, can you review screener on this one? The font size for the heading and description is much smaller than before. I followed Figma:
heading --1-wrap
description --2-wrap

**Pull request merged**